conf/config.py

```diff
@@ -20,6 +20,7 @@ class BaseConfiguration(object):
     SQLALCHEMY_DATABASE_URI = 'sqlite:///{0}'.format(os.path.join(
         dbdir, 'freshmaker.db'))
     SQLALCHEMY_TRACK_MODIFICATIONS = False
+    FRESHMAKER_ROOT_URL = "https://localhost"
```

Comment:
How about defining this in `TestConfiguration` instead?

Reply:
Sure